src/unix/bsd/freebsdlike/mod.rs

```diff
@@ -1593,6 +1593,14 @@ extern "C" {
     pub fn iconv_close(cd: iconv_t) -> ::c_int;
 }
 
+extern "C" {
```

Comment:
Why did you declare a new `extern` block instead of using the existing one?

Reply:
No particular reason I think. I started with glibc and there were already 3 different `extern "C"` blocks there (all without special `link` attributes), and I didn't know which one I should add this function to, so I added a `extern "C"` block there. And for the other platforms I copied my code from glibc.

Reply:
> 
> 
> No particular reason I think. I started with glibc and there were already 3 different `extern "C"` blocks there (all without special `link` attributes), and I didn't know which one I should add this function to, so I added a `extern "C"` block there. And for the other platforms I copied my code from glibc.

I should've overlooked them, let's use one block here.